packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useBreadcrumbsStyleProps } from './useBreadcrumbsStyleProps';
+import { SpiritBreadcrumbsProps } from '../../types';
+import { useStyleProps } from '../../hooks/styleProps';
+
+const defaultProps = {
+  goBackItemTranslation: 'Back',
+  items: [],
+};
+
+export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcrumbsProps<T>): JSX.Element => {
+  const { elementType: ElementTag = 'nav', goBackItemTranslation, items, ...restProps } = props;
+  const { classProps, props: modifiedProps } = useBreadcrumbsStyleProps({ ...restProps });
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  const isLast = (index: number) => {
+    return index === items.length - 1;
+  };
+
+  return (
+    <ElementTag
+      {...otherProps}
+      {...styleProps}
+      className={classNames(classProps, styleProps.className)}
+      aria-label="Breadcrumb"
+    >
+      <ol>
+        {items.map((item, index) => (
+          <React.Fragment key={`BreadcrumbsItem_${item.title}`}>
+            {index === 0 && (
+              <li className="d-tablet-none">
+                <a href={item.url}>{goBackItemTranslation}</a>
```

Comment:
Should we invent a better name for this prop, please? For example `goBackLabel` or `goBackTitle`? I think `goBackItemTranslation` is too long and there is no such thing as translating string so it can be misunderstood.

Reply:
`goBackTitle` was a bit unclear or on the other hand much simple for me, but if you're OK with that, I'm as well. 
It's been changed now.